resources/lang/pl/forum.php

```diff
@@ -258,8 +258,10 @@
         ],
 
         'watch' => [
-            'to_not_watching_done' => 'Usunięto subskrybcję tego wątku',
-            'to_watching_done' => 'Zasubskrybowano ten wątek',
+            'to_not_watching' => 'Nie subskrybuj',
```

Comment:
W oryginale jest "Not bookmarked", to na pewno nie będzie „Brak subskrypcji” albo „Nie zasubskrybowano”?